front/app/components/ConsentManager/Banner.tsx

```diff
@@ -135,18 +142,15 @@ const StyledCloseIconButton = styled(CloseIconButton)`
   transform: translateY(-50%);
   border: none;
   background: none;
-
```

Comment:
This was hiding the close button, probably as some kind of solution for it ending up below the 'Manage' button. The 40x right padding above solves that problem without hiding the close button